### PR TITLE
GH-65

### PR DESCRIPTION
**Pull request by gjvoosten**
Closes NCI-Agency/anet#65

### CODE REVIEW

File: client/src/pages/people/Form.js [hunk 1]
```diff
@@ -44,10 +45,12 @@ export default class PersonForm extends ValidatableFormWrapper {
 		const {ValidatableForm, RequiredField} = this
 
 		let willAutoKickPosition = person.status === 'INACTIVE' && person.position && !!person.position.id
+		let warnDomainUsername = person.status === 'INACTIVE' && person.domainUsername
```
Comment (gjvoosten) [inline client/src/pages/people/Form.js]: Sorry, this needs to change. Will update it!

File: client/src/pages/people/Form.js [hunk 2]
```diff
@@ -44,10 +45,12 @@ export default class PersonForm extends ValidatableFormWrapper {
...
 		let ranks = dict.lookup('ranks') || []
 
 		let currentUser = this.context.currentUser
 		let isAdmin = currentUser && currentUser.isAdmin()
+		let disableStatusChange = person.status === 'INACTIVE' || Person.isEqual(currentUser, person)
```
Comment (gjvoosten) [inline client/src/pages/people/Form.js]: Sorry, this needs to change. Will update it!

**Comment by gjvoosten** [comment]
@vassil I fixed my last commit, please review.

**Comment by dspCarlson** [comment]
NCI-Agency/anet#65: Don't let inactive people authenticate

Must also take into consideration that inactivated people leave and then come back a few months later or even years later.  The returning people will not necessarily have the same domain user name and/or email address.  We want to avoid creating multiple profiles of the same person.

**Pull request merged**